### PR TITLE
Tcfv2 support

### PR DESCRIPTION
**Pull request by leodoin**

### CODE REVIEW

File: app/src/main/java/com/sourcepoint/test_project/MainActivity.java [hunk 1]
```diff
@@ -36,6 +36,7 @@ private void removeView(View view) {
     private GDPRConsentLib buildGDPRConsentLib() {
         return GDPRConsentLib.newBuilder(22, "a-demo-property", 7055,"5c0e81b7d74b3c30c6852301",this)
                 .setStagingCampaign(false)
+                .setAuthId("17-02-2020-0001")
```
Comment (andresilveirah) [inline app/src/main/java/com/sourcepoint/test_project/MainActivity.java]: I believe we should remove the `authId`

File: cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/StoreClient.java [hunk 2]
```diff
@@ -48,36 +55,34 @@
...
-        editor.commit();
-    }
-
-    public void setIabConsentParsedPurposeConsents(String consentParsedPurposeConsents){
-        editor.putString(IAB_CONSENT_PARSED_PURPOSE_CONSENTS, consentParsedPurposeConsents);
+    public void setTCData(HashMap<String, String> data){
+        for(String s : data.keySet()){
+            editor.putString(s, data.get(s));
+        }
+        editor.putString(TC_KEYS_KEY, dataSetToStr(data.keySet()));
```
Comment (andresilveirah) [inline cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/StoreClient.java]: Why do we need the `TC_KEYS_KEY`?
So we can remove all IAB data later on?

Comment (leodoin) [reply cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/StoreClient.java]: Yes, and also in order to get the TCData back from the local storage.

File: app/src/main/java/com/sourcepoint/test_project/MainActivity.java (hunk 1 above)
Comment (leodoin) [reply app/src/main/java/com/sourcepoint/test_project/MainActivity.java]: indeed

**Pull request merged**